internal/supervisor/import.go

```diff
@@ -99,9 +105,80 @@ func NewImportSupervisor(awsClient toolsAWS.AWS, awat AWATClient, store installa
 	}
 }
 
+// completeImports transitions Installations back to stable after
+// checking with the AWAT to make sure that it has detected the
+// completion of the import
+func (s *ImportSupervisor) completeImports() error {
+	installationList, err := s.store.GetInstallations(
+		&model.InstallationFilter{
+			Paging: model.AllPagesNotDeleted(),
+			State:  model.InstallationStateImportComplete,
+		}, false, false)
+	if err != nil {
+		return errors.Wrap(err, "failed to list Installations")
+	}
+
+	for _, installation := range installationList {
```

Comment:
We probably want to work on the installations only once we have a valid lock. Not sure if we would have issues without that, but better safe than sorry.

Reply:
You're absolutely right

Reply:
Addressed in commit 3d6566d